sdk/storage/storage-file/tests.yml

```diff
@@ -0,0 +1,91 @@
+trigger: none
+
+jobs:
+  - template: ../../../eng/pipelines/templates/jobs/archetype-sdk-tests-node.yml
+    parameters:
+      PackageName: "@azure/storage-file"
+      EnvVars:
+        ACCOUNT_NAME: $(js-storage-test-account-name)
+        ACCOUNT_KEY: $(js-storage-test-account-key)
+        ACCOUNT_SAS: $(js-storage-test-account-sas)
+      Matrix:
+        Linux_Node8X:
```

Comment:
Feels like we are begging for storage specific template to avoid this duplication across the different storage yml files.

Reply:
Indeed